### PR TITLE
Update markedjs

### PR DESCRIPTION
**Pull request by fxha**
| Q                | A
| ---------------- | ---
| Bug fix?         | yes
| License          | MIT

### Description

- Updated markedjs to v0.6.1 (master) - [changelog](https://github.com/markedjs/marked/releases)
- Fixed missing emojis when exporting

I'll maybe improve some list and list item behavior later in other PR's. CommonMark 265 is already fixed but I'll open another PR when this PR is merged.

I have not changed our custom list implementation because upstream is more broken than our implementation, also not front matter, (inline) math and emojis. Furthermore, I merged your (@Jocs) task list changes with upstream (partially upstream and our code) for easier maintenance. I think I will open a few upstream issues due to list behavior.

### CODE REVIEW

File: src/muya/lib/parser/marked/renderer.js [hunk 1]
```diff
@@ -170,20 +161,11 @@ Renderer.prototype.del = function (text) {
...
-    }
-    if (prot.indexOf('javascript:') === 0 || prot.indexOf('vbscript:') === 0) {
-      return ''
-    }
+  href = cleanUrl(this.options.sanitize, this.options.baseUrl, href)
+  if (href === null) {
+    return text
   }
-  let out = '<a href="' + href + '"'
+  var out = '<a href="' + escape(href) + '"'
```
Comment (Jocs) [inline src/muya/lib/parser/marked/renderer.js]: If we want to be consistent with the code specification of markedjs, that is, use var to declare variables, then we need to ignore the marked folder in `eslintignore file`

Comment (fxha) [reply src/muya/lib/parser/marked/renderer.js]: I overlooked at, I'll replace all `var`'s.

**Comment by Jocs** [comment]
@fxha Pls update the `changelog`.

**Comment by Jocs** [comment]
@fxha great work!

**Comment by Jocs** [comment]
>I'll maybe improve some list and list item behavior later in other PR's. CommonMark 265 is already fixed but I'll open another PR when this PR is merged.

What's  CommonMark 265?

**Pull request merged**